### PR TITLE
Ensure that `requeueComputedKeyAndDecorators` is available

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | Fixes https://github.com/babel/babel/issues/16694
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

### CODE REVIEW

File: packages/babel-core/src/config/files/configuration.ts [hunk 1]
```diff
@@ -90,8 +90,7 @@ function* readConfigCode(
 
   // @ts-expect-error todo(flow->ts)
   if (typeof options.then === "function") {
-    // @ts-expect-error We use ?. in case options is a thenable
-    // but not a promise
+    // @ts-expect-error We use ?. in case options is a thenable but not a promise
```
Comment (nicolo-ribaudo) [inline packages/babel-core/src/config/files/configuration.ts:93]: This is just to trigger a babel/core release, as per https://github.com/babel/babel/issues/16694#issuecomment-2256979076

Comment (JLHwung) [reply packages/babel-core/src/config/files/configuration.ts:93]: Lol, I have a dedicate PR for that purpose: https://github.com/babel/babel/pull/16696

**Review by JLHwung — APPROVED** [review]

**Comment by PeachScript** [comment]
Hi @nicolo-ribaudo , when do you plant to release this?

**Comment by JLHwung** [comment]
> Hi @nicolo-ribaudo , when do you plant to release this?

Released in 7.25.2

**Pull request merged**